core/arch/arm/kernel/generic_entry_a64.S

```diff
@@ -92,8 +92,10 @@ copy_init:
 	 * Clear .bss, this code obviously depends on the linker keeping
 	 * start/end of .bss at least 8 byte aligned.
 	 */
-	adr	x0, __bss_start
-	adr	x1, __bss_end
+	adrp	x0, __bss_start
```

Comment:
Can we use a macro instead of writing the directly. I'm thinking of something like `mov_imm()`.

Reply:
Sure.